pkg/collector/python/datadog_agent.go

```diff
@@ -33,6 +34,15 @@ import (
 	"C"
 )
 
+type configFormat struct {
+	InitConfig interface{} `yaml:"init_config"`
+	Instances  []integration.RawMap
+}
+
+var (
+	checkRunner func(checkName string, configs []integration.Config)
```

Comment:
Is this really needed? Is this ever going to be something other than what you're currently setting it to?

Reply:
Because of circular dependencies I had to set it at runtime. It's very possible there is a better solution.